backend/api/v1/endpoints/Auth/OTP.py

```
@@ -0,0 +1,76 @@
+from fastapi import APIRouter, HTTPException, Depends
+from sqlalchemy.orm import Session
+from core.security import create_access_token, generate_otp, store_otp
+from fastapi.security import OAuth2PasswordBearer
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)